cmd/kube-controller-manager/app/core.go

```diff
@@ -23,6 +23,7 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"k8s.io/apimachinery/pkg/util/sets"
```

Comment:
Plz move this down, it should be placed together with k8s.io/*

Reply:
done for that